runbot_website_display/models/runbot_repo.py

```diff
@@ -9,5 +9,5 @@ class RunbotRepo(models.Model):
 
     _order = 'sequence asc'
 
-    sequence = fields.Integer()
+    sequence = fields.Integer(default=0)
```

Comment:
sure, but I'm in the middle of something else right now. If you've some time to spare, go ahead